benches/Cargo.toml

```diff
@@ -1,18 +1,24 @@
 [package]
-name = "fuel-core-benches"
-version = "0.0.0"
 edition = "2021"
 license = "BUSL-1.1"
+name = "fuel-core-benches"
 publish = false
+version = "0.0.0"
 
 [dependencies]
-fuel-core = { path = "../crates/fuel-core" }
+clap = { version = "4.0.17", features = ["derive"] }
```

Comment:
We likely shouldn't depend on patch versions here

Reply:
It's not pinned to `4.0.17` it just means `4.0.17` and above.

Reply:
We conventionally try to avoid setting patch version requirements, although it's less of a problem for binary targets. The main issue is that if libraries have different patch requirements it can cause headaches if they aren't compatible with each other when combined downstream.